src/main/java/smokefree/domain/Initiative.java

```diff
@@ -206,5 +216,46 @@ private void assertCurrentUserIsManager(MetaData metaData) {
                 "You are not a manager of this playground"));
     }
 
+    private void validateMaximumPlaygroundCapacity() {
+        if(initiativeProjection.playgrounds().size() >= initiativeProjection.MAXIMUM_PLAYGROUNDS_ALLOWED) {
+            throw new DomainException("MAXIMUM_PLAYGROUNDS_CAPACITY_REACHED",
+                    "Can not add more than " + initiativeProjection.MAXIMUM_PLAYGROUNDS_ALLOWED + " playgrounds",
```

Comment:
initiativeProjection.MAXIMUM_PLAYGROUNDS_ALLOWED don't put these constants in the projection. They should not have any validation related functionality.

Reply:
removed constants from the initiativeProjetion